app/scripts/modules/core/src/presentation/main.less

```diff
@@ -972,7 +972,7 @@ select:invalid {
     }
   }
   padding: 0;
-  border-bottom: 1px solid var(--color-silver);
+  border-top: 1px solid var(--color-silver);
```

Comment:
With a hybrid of the `CollapsibleSection` and `collapsibleSection.directive.html` some borders were on top and some were not. This makes them cohesive.